### PR TITLE
[BOARD_MEMBER] Add entities and fixtures

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: src/Entity/BoardMember/BoardMember.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @ORM\OneToOne(targetEntity="AppBundle\Entity\Adherent", inversedBy="boardMember")
+     * @ORM\JoinColumn(onDelete="CASCADE")
```
Comment (NyKo24) [inline src/Entity/BoardMember/BoardMember.php]: Are you sure ? a board member can keep Adherent status ?

File: src/Entity/BoardMember/BoardMember.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    }
+
+    public function setArea(string $area)
+    {
+        $this->area = $area;
+    }
+
+    public function addSkill(Skill $skill): void
+    {
+        if (!$this->skills->contains($skill)) {
```
Comment (NyKo24) [inline src/Entity/BoardMember/BoardMember.php]: skills is not a class attribute

File: src/Entity/BoardMember/BoardMember.php [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    /**
+     * @return Role[]|Collection|iterable
+     */
+    public function getRoles(): iterable
+    {
+        return $this->roles;
+    }
+
+    public function setRoless(ArrayCollection $roles): void
```
Comment (NyKo24) [inline src/Entity/BoardMember/BoardMember.php]: add "addRoles" function like "skills"

File: src/Entity/Adherent.php [hunk 4]
```diff
@@ -616,6 +628,23 @@ public function setCoordinatorManagedArea(CoordinatorManagedArea $coordinatorMan
...
+    public function setBoardMember(string $area, ArrayCollection $roles): void
+    {
+        $this->boardMember = new BoardMember();
+        $this->boardMember->setArea($area);
+        $this->boardMember->setRoles($roles);
+    }
+
+    public function isBoardMember(): bool
+    {
+        return $this->boardMember instanceof BoardMember && !empty($this->boardMember->getArea() && !empty($this->boardMember->getRoles()));
```
Comment (Remg) [inline src/Entity/Adherent.php]: `&& !empty($this->boardMember->getArea()) && !empty($this->boardMember->getRoles())`

File: src/DataFixtures/ORM/LoadAdherentData.php [hunk 5]
```diff
@@ -512,4 +518,11 @@ private function getCommitteeFactory(): CommitteeFactory
     {
         return $this->container->get('app.committee.factory');
     }
+
+    public function getDependencies()
+    {
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadAdherentData.php]:
```
return [
   // ...
];
```

File: src/DataFixtures/ORM/LoadAdherentSubscriptionData.php [hunk 6]
```diff
@@ -32,4 +33,11 @@ public function load(ObjectManager $manager)
 
         $manager->flush();
     }
+
+    public function getDependencies()
+    {
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadAdherentSubscriptionData.php]: same as above

File: src/DataFixtures/ORM/LoadCitizenInitiativeData.php [hunk 7]
```diff
@@ -264,8 +264,11 @@ private function getEventRegistrationFactory(): EventRegistrationFactory
         return $this->container->get('app.event.registration_factory');
     }
 
-    public function getOrder()
+    public function getDependencies()
     {
-        return 2;
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadCitizenInitiativeData.php]: same as above

File: src/DataFixtures/ORM/LoadEventData.php [hunk 8]
```diff
@@ -367,8 +367,11 @@ private function getEventRegistrationFactory(): EventRegistrationFactory
         return $this->container->get('app.event.registration_factory');
     }
 
-    public function getOrder()
+    public function getDependencies()
     {
-        return 2;
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadEventData.php]: same as above

File: src/DataFixtures/ORM/LoadProcurationData.php [hunk 9]
```diff
@@ -218,4 +219,11 @@ public function load(ObjectManager $manager)
 
         $manager->flush();
     }
+
+    public function getDependencies()
+    {
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadProcurationData.php]: same as above

File: src/DataFixtures/ORM/LoadReferentManagedUserData.php [hunk 10]
```diff
@@ -135,8 +135,10 @@ private function getReferentManagedUserFactory(): ReferentManagedUserFactory
         return new ReferentManagedUserFactory();
     }
 
-    public function getOrder()
+    public function getDependencies()
     {
-        return 5;
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadReferentManagedUserData.php]: same as above

File: src/DataFixtures/ORM/LoadSummaryData.php [hunk 11]
```diff
@@ -207,8 +207,11 @@ private function getSummaryFactory(): SummaryFactory
         return new SummaryFactory(new Slugify());
     }
 
-    public function getOrder()
+    public function getDependencies()
     {
-        return 3;
+        return array(
```
Comment (Remg) [inline src/DataFixtures/ORM/LoadSummaryData.php]: same as above

**Pull request merged**